### ISSUE TITLE
Hide console

### ISSUE THREAD

**Issue by ransomts** [StevensNJD4/LazyMan#17]
When I just want to watch a game, I don't super care about seeing the info the console prints out.
It'd be nice to have an option in preferences to hide all that. Probably wouldn't be hard to just disable
the panel the console lives in ;)

**Comment by StevensNJD4** [comment]
i kept it there so users will know lazyman is doing something when they click play

**Comment by ransomts** [comment]
how would you feel about a bottom status bar instead? Current console output could still be logged to a file if you like that for debugging.

Something like this:
[Downloading: amount retrieved 1.3GB]
[Error: Connecting to Server]
[Streaming: Flyers @ Hurricanes <game time>]

Maybe just some spinning wheel to show network activity or a loading bar to show download progress or streaming game percentage? idk I'm just throwing out ideas and want to see if they're wanted before I make a PR for something more serious than aligning panels.

**Comment by StevensNJD4** [comment]
Or we can copy nhlgames and put it on a separate tab. Then have a status bar

**Comment by ransomts** [comment]
That can work yeah, name the extra tab 'status' or something since console implys you can enter commands.

**Comment by MrAureliusR** [comment]
I have to say, the new theme looks good, but hiding the console definitely needs to be an option. I've had it happen a few times now where the console squishes everything upwards and you can only see a single game row at a time. If it happens again I will screenshot (seems to happen when I save stream).

**Comment by MrAureliusR** [comment]
Yes, exactly, though mine will often end up even skinnier than that, barely taller than a single game. My resolution is only 1600x900 though. I DEFINITELY **love** the new theme and the calendar especially! If I knew anything other than C for embedded devices I would try and help out, unfortunately I'm useless with GUI stuff.

**Comment by ransomts** [comment]
Sure thing, I'll get on this soon as I can

**Comment by StevensNJD4** [comment]
@ransomts can you please fix the ui so the checkboxes appear in linux and osx?

**Comment by ransomts** [comment]
I'll take a crack for linux but I don't have an apple machine to test the osx on. Commit incoming

**Comment by ransomts** [comment]
I'm having a problem finding which checkboxes aren't appearing, can I get a screenshot or something? @StevensNJD4

**Comment by inickt** [comment]
![image](https://user-images.githubusercontent.com/4574041/48102244-2db86b80-e1f8-11e8-9758-e5065ee0afbc.png)

Here is what it looks like on Mac when initially opening the program. Guessing it is the same for Linux. You have to drag the window to be wider for the checkboxes to appear.

**Comment by inickt** [comment]
Just wanted to point out one more thing– really small but I noticed it nonetheless. It looks like the left column for Away teems is centered vertically, and does not align, while the right column for home teams is left justified and all line up.

**Comment by ransomts** [comment]
I think the alignment issue is caused by the differing logo png widths, I'm kinda surprised this isn't happening on windows as well.

I can't think of an easy way to fix this at runtime, but it would probably be at [https://github.com/StevensNJD4/LazyMan/blob/master/src/Util/IconTextCellRemderer.java#L23](url)

Or we can just pad the png files to give them a consistent width, have a preference @StevensNJD4 ?

**Comment by StevensNJD4** [comment]
let's pad em
